imls-raspberry-pi/internal/state/session.go

```diff
@@ -0,0 +1,33 @@
+package state
+
+import (
+	"time"
+)
+
+type sessionId struct {
+	id int64
+}
+
+var (
+	// singleton pattern. not thread safe.
+	currentSession = sessionId{0}
```

Comment:
egh. not totally happy with this, but it's the best short-term solution i could come up with. ideas, suggestions for improvement are ofc welcomed

Reply:
This looks like the kind of code I had in the transition. We might want to do a once-over code walk, and see if there's a refactor that feels obvious... but adding a few functions to `state` was my solution in another instance as well. Perhaps not ideal, but we ended up with *so much less code* by switching to Viper, I'm not worried about the handful of helpers to manage some state/simplify the transition.